Sources/Core/Shared/Logging.swift

```diff
@@ -299,7 +299,7 @@ public extension NSOperation {
      set, this resorts to the class description.
     */
     var operationName: String {
-        return name ?? "\(self)"
+        return name ?? "<unnamed operation>"
```

Comment:
I think maybe just `Unnamed Operation` will be better here. I tend to use the `<T>` when we have generic types involved.